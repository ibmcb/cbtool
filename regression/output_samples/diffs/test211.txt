|host_simhostd0                |simzone_d           |hypervisor                                        |LG           |simhostd0                   |157.232.5.229   
|host_simhostd4                |simzone_d           |hypervisor                                        |LG           |simhostd4                   |101.60.8.185    
|host_simhostd2                |simzone_d           |hypervisor                                        |LG           |simhostd2                   |184.97.156.191  
|host_simhostd3                |simzone_d           |hypervisor                                        |LG           |simhostd3                   |200.194.32.107  
|host_simhostd1                |simzone_d           |hypervisor                                        |LG           |simhostd1                   |207.82.22.231   
|host_simhostb2                |simzone_b           |hypervisor                                        |SUT          |simhostb2                   |139.240.109.118 
|host_simhostb0                |simzone_b           |hypervisor                                        |SUT          |simhostb0                   |76.63.10.163    
|host_simhostb1                |simzone_b           |hypervisor                                        |SUT          |simhostb1                   |95.83.132.61    
|host_simhostb3                |simzone_b           |hypervisor                                        |SUT          |simhostb3                   |116.57.214.197  
|host_simhostb4                |simzone_b           |hypervisor                                        |SUT          |simhostb4                   |21.219.64.143   
|host_simhosta0                |simzone_a           |hypervisor                                        |SUT          |simhosta0                   |24.169.15.34    
|host_simhosta3                |simzone_a           |hypervisor                                        |SUT          |simhosta3                   |144.137.200.103 
|host_simhosta1                |simzone_a           |hypervisor                                        |SUT          |simhosta1                   |160.46.76.168   
|host_simhosta2                |simzone_a           |hypervisor                                        |SUT          |simhosta2                   |220.240.227.169 
|host_simhosta4                |simzone_a           |hypervisor                                        |SUT          |simhosta4                   |98.114.129.145  
|host_simhostc0                |simzone_c           |hypervisor                                        |SUT          |simhostc0                   |179.20.229.230  
|host_simhostc2                |simzone_c           |hypervisor                                        |SUT          |simhostc2                   |160.116.76.235  
|host_simhostc1                |simzone_c           |hypervisor                                        |SUT          |simhostc1                   |14.94.116.120   
|host_simhostc3                |simzone_c           |hypervisor                                        |SUT          |simhostc3                   |147.23.39.17    
|host_simhostc4                |simzone_c           |hypervisor                                        |SUT          |simhostc4                   |57.234.83.225   
