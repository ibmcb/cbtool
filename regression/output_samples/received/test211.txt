The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |223.57.181.60   |simhostb3                  |SUT                 |none    |none       |871DBCA3-6BD3-51CC-AD1C-EC170141AF08

