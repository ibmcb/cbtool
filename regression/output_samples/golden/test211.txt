The following HOSTs are attached to this experiment (Cloud TESTCLOUD) :
|name                          |vmc_name            |function                                          |pool         |cloud_hostname              |cloud_ip        
|host_simhostb4                |simzone_b           |hypervisor                                        |SUT          |simhostb4                   |26.245.59.172   
|host_simhostb2                |simzone_b           |hypervisor                                        |SUT          |simhostb2                   |53.170.8.16     
|host_simhostb3                |simzone_b           |hypervisor                                        |SUT          |simhostb3                   |76.13.160.104   
|host_simhostb0                |simzone_b           |hypervisor                                        |SUT          |simhostb0                   |64.27.138.178   
|host_simhostb1                |simzone_b           |hypervisor                                        |SUT          |simhostb1                   |121.63.29.216   
|host_simhostc3                |simzone_c           |hypervisor                                        |SUT          |simhostc3                   |118.253.143.122 
|host_simhostc4                |simzone_c           |hypervisor                                        |SUT          |simhostc4                   |180.175.140.246 
|host_simhostc1                |simzone_c           |hypervisor                                        |SUT          |simhostc1                   |174.182.6.130   
|host_simhostc0                |simzone_c           |hypervisor                                        |SUT          |simhostc0                   |217.37.153.191  
|host_simhostc2                |simzone_c           |hypervisor                                        |SUT          |simhostc2                   |154.230.138.196 
|host_simhosta2                |simzone_a           |hypervisor                                        |SUT          |simhosta2                   |250.167.11.199  
|host_simhosta4                |simzone_a           |hypervisor                                        |SUT          |simhosta4                   |179.164.6.116   
|host_simhosta0                |simzone_a           |hypervisor                                        |SUT          |simhosta0                   |215.99.128.185  
|host_simhosta3                |simzone_a           |hypervisor                                        |SUT          |simhosta3                   |230.146.159.104 
|host_simhosta1                |simzone_a           |hypervisor                                        |SUT          |simhosta1                   |86.116.69.246   
|host_simhostd4                |simzone_d           |hypervisor                                        |LG           |simhostd4                   |118.105.212.224 
|host_simhostd3                |simzone_d           |hypervisor                                        |LG           |simhostd3                   |214.64.204.230  
|host_simhostd0                |simzone_d           |hypervisor                                        |LG           |simhostd0                   |194.184.30.95   
|host_simhostd1                |simzone_d           |hypervisor                                        |LG           |simhostd1                   |230.35.175.46   
|host_simhostd2                |simzone_d           |hypervisor                                        |LG           |simhostd2                   |8.72.196.162    

